.github/workflows/publish-release.yml

```diff
@@ -14,11 +14,11 @@ jobs:
         with:
           token: ${{ secrets.WSO2_INTEGRATION_BOT_TOKEN }}
           fetch-depth: 0
-      - name: Set up JDK 11
+      - name: Set up JDK 17
         uses: joschi/setup-jdk@v2
         with:
           distribution: 'adopt'
-          java-version: 11
+          java-version: 17
```

Comment:
Since we support Java 11, will this cause runtime issues?

Reply:
The plugin versions used require Java 17 to build this repository.